.controlplane/controlplane.yml

```diff
@@ -0,0 +1,18 @@
+aliases:
+  common: &common
+    org: shakacode-staging
+    location: aws-us-east-2
+    one_off_workload: rails
+    app_workloads:
+      - rails
+    additional_workloads:
+      - redis
+      - postgres
```

Comment:
@dzirtusss why do we nee to list these?

Reply:
those are used in `ps` command atm - `ps`, `ps:start`, `ps:stop`, `ps:restart`. however, there is no clear plan about their future need atm